File: UI/city-demo/src/app/cities/pagination.model.ts
export class PaginationModel {
    pageNumber : number;
    pageSize : number;
    totalPages : number;
    totalRecords : number;

    constructor(pageNumber: number, pageSize: number, totalPages: number, totalRecords: number) {
        this.pageNumber=pageNumber;
        this.pageSize=pageSize;
        this.totalPages=totalPages;
        this.totalRecords=totalRecords;
    }


}

File: Back-End/City-Img-Processing/src/main/java/com/hpu/city/model/PaginationModel.java
package com.hpu.city.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

/**
 * Class represents the PaginationModel which is used for communication over the rest layer
 */
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class PaginationModel {
    int pageNumber;
    int pageSize;
    int totalPages;
    long totalRecords;
}
